client/src/components/Login.jsx

```diff
@@ -1,7 +1,9 @@
 import React, { useState } from "react";
 import { Link } from "react-router-dom";
-import { makeStyles, Button, TextField, Grid } from "@material-ui/core";
+import { makeStyles, TextField, Grid } from "@material-ui/core";
+
 import LoginSignupContainer from "./LoginSignupContainer";
+import SubmitButton from "./ui/SubmitButton";
```

Comment:
I don't love the folder name ui - but if you see it commonly used, then that is fine 

I'd actually prefer that we have a folder called components for these UI elements, and that this folder components is renamed to views or pages 

the components folder also wouldn't be nested in the views or pages folder